1.2/spec.md

```diff
@@ -975,35 +1002,28 @@ Depending on the particular HTML markup used in the document, the document is
 suitable for different kinds of processing and use. The formats have the
 following intents:
 
-## `html_none`
-
-Straightforward equivalent of Goodoc or [[XDOC]]
-
-## `html_ocr`
-
-Straightforward recording of commercial OCR system output
-
-## `html_absolute`
-
-Target format for services like Google's View as HTML
-
-## `html_xytable`
-
-Target format for layout-preserving on-screen document viewing
-
-## `html_simpl`
-
-Target format for convenient on-line viewing and intermediate format for indexing
+  : <a>html_none</a> (see [[#format-none]])
+  :: Straightforward equivalent of Goodoc or [[XDOC]]
+  : <a>html_simple</a>
+  :: Target format for convenient on-line viewing and intermediate format for indexing
+  : <a>html_xytable_absolute</a>, <a>html_xytable_relative</a>
+  :: Target format for layout-preserving on-screen document viewing
+  : Formats defined in [[#format-ocr]]
+  :: Straightforward recording of commercial OCR system output
+  : Formats defined in [[#format-absolute]]
+  :: Target format for services like Google's View as HTML
 
 As long as a format contains the hOCR information, it can be reprocessed by
 layout analysis software and converted into one of the other formats. In
 particular, we envision layout analysis tools for converting any hOCR document
-into `html_absolute`, `html_xytable`, and `html_simple`. Furthermore,
-internally, a layout analysis system might use `html_xytable` as an
-intermediate format for converting hOCR into `html_simple`.
+into <a>html_absolute</a>, <a>html_xytable_absolute</a>, and
```

Comment:
This link to "html_absolute" is not working. I guess because the anchor does no exists anymore.